src/components/SpaceProposalPage.vue

```diff
@@ -143,6 +131,8 @@ onMounted(() => setMessageVisibility(props.proposal.flagged));
 
       <template v-else>
         <div class="px-3 md:px-0">
+          <LabelProposalState :state="proposal.state" class="mb-[12px]" />
```

Comment:
Shouldn't this be more relevant inside `SpaceProposalHeader`, like before ?

Reply:
Maybe, but the header is too cramped (especially on mobile), and I think it makes sense the way the designers solved it

Reply:
I get the UI changes. My concern is that this component belongs to `SpaceProposalHeader` logically, and moving it inside that component will not change the UI

Reply:
Oh I see, will look into it

Reply:
Doesn't work because it needs to be outside of the layout and above the message so the user have go back when they visit a flagged proposal